docs/src/main/asciidoc/spring-cloud-commons.adoc

```diff
@@ -1228,6 +1228,25 @@ NOTE: The meters are registered in the registry when at least one record is adde
 
 TIP: You can further configure the behavior of those metrics (for example, add https://micrometer.io/docs/concepts#_histograms_and_percentiles[publishing percentiles and histograms]) by https://docs.spring.io/spring-boot/docs/current/reference/html/production-ready-features.html#production-ready-metrics-per-meter-properties[adding `MeterFilters`].
 
+=== Spring Cloud LoadBalancer per client configuration
+You can configure multiple loadBalancer clients by using `spring.cloud.loadbalancer.client.services[]` property:
```

Comment:
Please add a few more words of explanation indicating that the properties can have different values per client, that there's a switch flag to enable this function and that, by default, the global properties only are being used to avoid backwards incompatibility.